deploy/docker-compose.yml

```diff
@@ -68,7 +68,6 @@ services:
       - AUTH=DISABLE
     ports:
       - 8443:443
-      - 8080:80
```

Comment:
This is might be a required change, but I do not think it should be covered by your change request, right?

Reply:
Yes, I changed this for local testing and I will remove this change.

Reply:
rollback the change